src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -597,9 +598,45 @@ private async Task<Dictionary<string, string>> PrepareData(Dictionary<string, Co
             // Reset old feature flag tracing in order to track the information present in the current response from server.
             _options.FeatureFlagTracing.ResetFeatureFlagTracing();
 
```

Comment:
Seems like we should have ResetAiConfigurationTracing(); Like we have ResetFeatureFlagTracing above.